src/utils/debounce/index.ts

```diff
@@ -0,0 +1,8 @@
+export const debounce = (callback: Function, delay: number) => {
+  let timeout: number
+
+  return () => {
+    clearTimeout(timeout)
+    timeout = setTimeout(callback, delay)
+  }
+}
```

Comment:
Hmm. I know that we're already having something inside Sirius for debouncing (`useDebounce`). Might be nice to move this to Solar, so we can have one dedicated util/hook for debouncing, and not 1 in Solar and 1 in Sirius 😬